test/basic.js

```diff
@@ -53,6 +53,24 @@ test('set', function (t) {
   })
 })
 
+test('set ipv6', function (t) {
+  t.plan(4)
+  hostile.set('::1', 'peercdn.com', function (err) {
+    t.error(err)
+    hostile.get(false, function (err, lines) {
+      t.error(err)
+      var exists = lines.some(function (line) {
+        return line[0] === '::1' && line[1] === 'peercdn.com'
+      })
+      t.ok(exists, 'ipv6 line was not added')
```

Comment:
The message here should describe what you're checking for, not what happened in the failure case.